lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -265,8 +265,8 @@ def add_copy_resources_script_phase
             unless resource_paths_by_config.values.all?(&:empty?)
               input_paths = [target.copy_resources_script_relative_path, *resource_paths_by_config.values.flatten.uniq]
               output_paths = ['${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}']
+              TargetIntegrator.add_copy_resources_script_phase_to_target(native_target, script_path, input_paths, output_paths)
```

Comment:
what happens if a target had some resources or framework paths, but then they were removed, maybe because the pod was removed. Will the script be updated to denote this?

Reply:
Ah interesting, I tried the other way around and it was successfully added, but I'll test that case too

Reply:
I think this comment is the only real blocker

Reply:
Working on this, and adding test coverage, tomorrow

Reply:
Awesome!! Thanks!!